conftest.py

```diff
@@ -32,8 +32,14 @@
 
 AIRFLOW_COMPONENT_CACHE_INSTANCE = {
     "display_name": "Airflow Example Components",
-    "metadata": {"runtime_type": "APACHE_AIRFLOW", "categories": ["examples"]},
-    "schema_name": "elyra-airflow-examples-catalog",
+    "metadata": {
+        "runtime_type": "APACHE_AIRFLOW",
+        "categories": ["examples"],
+        "paths": [
+            "https://raw.githubusercontent.com/elyra-ai/examples/master/component-catalog-connectors/airflow-example-components-connector/airflow_examples_connector/resources/bash_operator.py"  # noqa
```

Comment:
Confirmed that the tests are passing with the legacy resource URL:

![image](https://user-images.githubusercontent.com/13068832/161787213-3dad8029-80e1-4ee6-a744-dbec603be610.png)

We should now be able to switch to the new resource, as mentioned in https://github.com/elyra-ai/elyra/pull/2636#issue-1193283831